roles/openshift_logging_fluentd/defaults/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+### General logging settings
+openshift_logging_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
+openshift_logging_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_image_pull_secret: "{{ openshift_hosted_logging_image_pull_secret | default('') }}"
+openshift_logging_master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
+openshift_logging_master_public_url: "{{ openshift_hosted_logging_master_public_url | default('https://' + openshift.common.public_hostname + ':' ~ (openshift_master_api_port | default('8443', true))) }}"
+openshift_logging_namespace: logging
+openshift_logging_tempdir: ''
+
```

Comment:
The intent is that each component role should be self contained so there will be duplication. The names should likely be more specific in each component role. I will also have the current `openshift_logging` role setting those values when it calls the roles. So the user's inventory could look the same as it does now